sdk/cosmos/microsoft-azure-cosmos/src/main/java/com/azure/data/cosmos/internal/http/ReactorNettyClient.java

```diff
@@ -118,8 +117,7 @@ private void configureChannelPipelineHandlers() {
                 reactorNettyRequest.header(header.name(), header.value());
             }
             if (restRequest.body() != null) {
-                Flux<ByteBuf> nettyByteBufFlux = restRequest.body().map(Unpooled::wrappedBuffer);
-                return reactorNettyOutbound.send(nettyByteBufFlux);
+                return reactorNettyOutbound.send(restRequest.body());
```

Comment:
interesting. we missed this :-)

great finding.

Reply:
Thanks Mo :) 
Thanks to @anuchandy  as well for the help so far :)